openpathsampling/deprecations.py

```diff
@@ -204,6 +204,20 @@ def version_tuple_to_string(version_tuple):
     deprecated_in=(1, 6, 0)
 )
 
+NEW_DEFAULT_FILENAME_SETTER =  Deprecation(
+    problem=("The default FilenameSetter for external engines is now a counter"
+             ", but will become a random string. This is more robust against "
+             "accidental overwrites."),
+    remedy=("If you want to keep the old counting behavior, add "
+            '{"filename_setter": '
+            "paths.engines.external_engine.FilenameSetter() } to the 'options'"
+            " of this engine. Otherwise, this behaviour will automatically "
```

Comment:
```suggestion
            " of this engine. Otherwise, this behavior will automatically "
```

we spell the 'Murican way here! 😂